src/graphql/getter.ts

```diff
@@ -165,17 +158,17 @@ export default class Getter extends CommandBase {
     return this;
   };
 
-  withLimit = (limit: any) => {
+  withLimit = (limit: number) => {
     this.limit = limit;
     return this;
   };
 
-  withOffset = (offset: any) => {
+  withOffset = (offset: number) => {
     this.offset = offset;
     return this;
   };
 
-  withSort = (sortObj: any) => {
+  withSort = (sortObj: object) => {
```

Comment:
Do we have a type (or interface) for `Sort`?

Reply:
Updated with interface for Sort